### PR TITLE
Add User Profile view

### PR DESCRIPTION
**Pull request by hbalenda**

### CODE REVIEW

File: app/helpers/users_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (brittanyjspringer) [inline app/helpers/users_helper.rb]: Won't be using this, so let's delete.

File: app/assets/javascripts/users.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (brittanyjspringer) [inline app/assets/javascripts/users.coffee]: Delete, please.

File: app/assets/stylesheets/users.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (brittanyjspringer) [inline app/assets/stylesheets/users.scss]: Delete, please.

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<h1>Profile</h1>
+<p>Email: <%= @user.email %></p>
+<p>Getting things done since <%= @user.created_at.strftime("%b %d, %Y") %></p>
```
Comment (brittanyjspringer) [inline app/views/users/show.html.erb:3]: I LOVE that. A++++

File: spec/helpers/users_helper_spec.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (brittanyjspringer) [inline spec/helpers/users_helper_spec.rb]: Delete, please.

File: spec/views/users/show.html.erb_spec.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (brittanyjspringer) [inline spec/views/users/show.html.erb_spec.rb]: Delete, please.

**Comment by brittanyjspringer** [comment]
@hcbviolet Just some file cleanup and we'll be good to merge. Please @ me here when it is ready. Thanks!

**Comment by hbalenda** [comment]
@Brit200313 All done! I'm hoping you meant to delete the whole files for those :dizzy_face: . I'm sure I can add anything back that needs to be.

**Comment by brittanyjspringer** [comment]
Yes, delete the whole thing!

**Pull request merged**